src/main.ts

```diff
@@ -160,15 +160,15 @@ async function haltPullRequest(
   context: Context,
   inputs: Inputs,
   pullRequest: PullRequest,
-  message: Message,
+  msg: Message,
 ): Promise<void> {
   console.info(`Setting halted status for PR #${pullRequest.number}`);
   await githubApi.createCommitStatus(client, {
     ...context.repo,
     sha: pullRequest.head.sha,
     context: inputs.statusContext,
     state: "failure",
-    description: message.title,
+    description: message.toStatusDescription(msg),
```

Comment:
Where is `message` coming from? Is it some kind of global variable? 

Is this more appropriate?

```suggestion
    description: msg.toStatusDescription(msg),
```

Reply:
This module has `import * as message from './message'`. This function was previously shadowing the `message` module as a `message` variable. Once I tried to use `message.toStatusDescription()` that became a problem because `message` (the variable) didn't have the module's function. Fixing the shadowing (by using `msg` for the variable at L163) gave me back `message` the module to use.

I think if this project had ESLint configured, the shadowing would've been caught before this.